addons/account/models/ir_actions_report.py

```diff
@@ -45,3 +48,11 @@ def render_qweb_pdf(self, res_ids=None, data=None):
                     raise UserError(_("Only invoices could be printed."))
 
         return super().render_qweb_pdf(res_ids=res_ids, data=data)
+
+    def _get_rendering_context(self, docids, data):
+        data = data and dict(data) or {}
+        ids = isinstance(docids, int) and [docids] or docids
+        invoice = self.env['account.move'].search([('id', 'in', ids)])
```

Comment:
browse?

Reply:
invoice.mapped('currency_id.rounding') return an error if the record doesn't exists, with search it return an empty list instead

Reply:
If which record doesn't exist? How is that possible?

Reply:
_get_rendering_context is not only trigger with account moves apparently